src/js/yeldaWebchatInjector.js

```diff
@@ -168,16 +169,15 @@ class YeldaChat {
   }
   /**
    * Load CSS asynchroneously
-   * @param {String} origin domain url
   */
 
-  loadCssAsync (origin) {
+  loadCssAsync () {
     const head = document.getElementsByTagName('head')[0]
     const yeldaCss = document.createElement('link')
     yeldaCss.rel = 'stylesheet'
     yeldaCss.type = 'text/css'
     yeldaCss.crossorigin = 'anonymous'
-    yeldaCss.href = origin + '/yelda/css/injector.min.css'
+    yeldaCss.href = '/node_modules/yelda-webchat/dist/css/yeldaWebchatInjector.min.css'
```

Comment:
why not a more relative path, like ../js/.. ?

Reply:
Yeah, we can't use node_modules path to load the css, now I have changed it to load from assistant url with correct css path